oracle/unlock_sample_data.sql

```diff
@@ -0,0 +1,7 @@
+alter user hr account unlock;
+/
+
+alter user hr identified by oracle;
```

Comment:
Can we set this password as a parameter in `config.properties` and then pass it in as a parameter to this script?